dpnp/dpnp_algo/dpnp_algo.pyx

```diff
@@ -140,6 +142,20 @@ cpdef dparray dpnp_astype(dparray array1, dtype_target):
     return result
 
 
+cpdef dparray dpnp_flatten(dparray self):
```

Comment:
rename `self` param here. Avoid using keywords for variables.
use `array_` instead of `self` here.

This was proposed before.